runtime/src/iree/builtins/ukernel/mmt4d_tile_generic.c

```diff
@@ -0,0 +1,120 @@
+// Copyright 2022 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/builtins/ukernel/mmt4d_tile_generic.h"
+
+// In order to be helpful as a reference for future architecture-specific
```

Comment:
this is cool! often in reference implementations they're implemented so differently from the optimized ones that it's hard to correlate the two - this should be much easier for people to take and translate than needing to look at a different arch-specific implementation and translate that

Reply:
yup, this kind of meta issue is somewhat important here, the hope being that over time this will get ported by to other archs by other people.